src/backy/main.py

```diff
@@ -308,32 +397,39 @@ def setup_argparser():
     )
     p.set_defaults(func="forget")
 
-    return parser
+    return parser, client
 
 
 def main():
-    parser = setup_argparser()
+    parser, client_parser = setup_argparser()
     args = parser.parse_args()
 
     if not hasattr(args, "func"):
         parser.print_usage()
         sys.exit(0)
+    if args.func == "client" and not hasattr(args, "apifunc"):
+        client_parser.print_usage()
+        sys.exit(0)
 
     if not hasattr(args, "logfile"):
         args.logfile = None
 
-    is_daemon = args.func == "scheduler" or args.func == "check"
-    default_logfile = (
-        Path("/var/log/backy.log")
-        if is_daemon
-        else args.backupdir / "backy.log"
-    )
+    match args.func:
+        case "scheduler":
+            default_logfile = Path("/var/log/backy.log")
+        case "client":
+            default_logfile = None
+        case _:
+            default_logfile = args.backupdir / "backy.log"
 
     # Logging
     logging.init_logging(
         args.verbose,
         args.logfile or default_logfile,
-        default_job_name="-" if is_daemon else "",
+        default_job_name="-"
```

Comment:
I think this warrants bracing to help the indentation. This looks a bit weird because the line before the if seems complete.

Reply:
Alternatively, extract the default_job_name into a match argument as you did with the default logfile ...